src/iap.ts

```diff
@@ -206,13 +162,11 @@ export const getProducts = <SkuType extends string>(
 export const getSubscriptions = (skus: string[]): Promise<Subscription[]> =>
   (
     Platform.select({
-      ios: async () => {
-        checkNativeiOSAvailable();
-
-        return RNIapIos.getItems(skus).then((items: Subscription[]) =>
+      ios: getIosModule()
+        .getItems(skus)
+        .then((items: Subscription[]) =>
```

Comment:
The issue raised in current lines of code #1454. However, I am not sure how this breaks.

I'll look in to it.

@mathmania @andresesfm